oak_runtime/introspection_browser_client/build

```diff
@@ -2,6 +2,25 @@
 #
 # Build browser introspection client.
 
+#######################################
+# Copies a directory without overwriting identical files at the destination.
+# Arguments:
+#   A source directory, a destination directory.
+#######################################
+function copy_directory_nondestructively() {
```

Comment:
BTW, I wonder if this is essentially equivalent to `rsync --checksum`? Cc @daviddrysdale